wire-library/wire-reflector/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=wire-reflector
+POM_NAME=Wire
+POM_DESCRIPTION=Reflector
```

Comment:
Maybe something more speaking? It can be a sentence.

Reply:
Updated.